DataFormats/L1THGCal/interface/HGCalCluster.h

```diff
@@ -13,7 +13,9 @@ namespace l1t {
     
     public:
        
-      HGCalCluster(){}
+      HGCalCluster(){
+        this->setClusterStep(ClusterStep::FirstLayer);
```

Comment:
It should probably not be hardcoded like that in the constructor since we may want to have `HGCalCluster` produced by the `SecondLayer` in some architecture. It should be the algorithm that set the `ClusterStep` of the cluster it produced.